docs/use/ssh.rst

```diff
@@ -0,0 +1,43 @@
+Connecting to servers with SSH
+==============================
+
+.. note::
+    Before you can SSH into the OCP servers you first need to upload your SSH public key.
+    `See notes on salt install page. <https://ocdsdeploy.readthedocs.io/en/latest/develop/get_started.html?highlight=ssh#add-public-key-to-remote-servers>`_. 
```

Comment:
Add a `.. _add-public-key:` anchor on `get_started.rst` and then use:

```rst
See :ref:`add-public-key`.
```

Reply:
Fixed in commit aacfdedd99b3400c3a80aa60250ef01c18831650